Description: autotools: rationalize: README.me
 Minor upstream fix: README.me is not present in the distributed source ball,
 but README is.
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2016-02-20

--- a/Makefile.am
+++ b/Makefile.am
@@ -17,14 +17,15 @@
 
 EXTRA_DIST = autogen.sh README
 
-if MAINTAINER_MODE
-
-all-local: README
-
-README: README.md
-	sed -e '/^Upgrade Notes$$/p' -e '/^Git$$/,/^Upgrade Notes$$/d' $< > $@
-
-endif
+## README.md not present in upstream source tarball (3.0.0)
+##if MAINTAINER_MODE
+##
+##all-local: README
+##
+##README: README.md
+##	sed -e '/^Upgrade Notes$$/p' -e '/^Git$$/,/^Upgrade Notes$$/d' $< > $@
+##
+##endif
 
 uninstall-local:
 	@-rmdir --ignore-fail-on-non-empty $(DESTDIR)$(docdir)
